services/src/main/java/org/keycloak/services/clientregistration/policy/impl/TrustedHostClientRegistrationPolicy.java

```diff
@@ -97,29 +97,34 @@ protected void verifyHost() throws ClientRegistrationPolicyException {
 
         String hostAddress = session.getContext().getConnection().getRemoteAddr();
 
-        verifyHost(hostAddress);
-    }
+        logger.debugf("Verifying remote host : %s", session.getContext().getConnection().getRemoteHost());
```

Comment:
Similar question as above. Why to log host if what we're really checking is the IP?

Reply:
For display / logging purposes we're using the host.